src/layers.jl

```diff
@@ -108,19 +125,32 @@ end
 
 @functor Patching
 
-"""
-    mlpblock(planes, expansion_factor = 4, dropout = 0., dense = Dense)
+# Positional embedding layer used by many vision transformer-like models
+struct PosEmbedding
+  embedding_vector
+end
 
-Feedforward block used in many vision transformer-like models.
+(p::PosEmbedding)(x) = x .+ p.embedding_vector[:, 1:size(x)[2], :]
 
-# Arguments
-  `planes`: Number of dimensions in the input and output.
-  `hidden_planes`: Number of dimensions in the intermediate layer.
-  `dropout`: Dropout rate.
-  `dense`: Type of dense layer to use in the feedforward block.
-  `activation`: Activation function to use.
-"""
-function mlpblock(planes, hidden_planes, dropout = 0., dense = Dense; activation = gelu)
-  Chain(dense(planes, hidden_planes, activation), Dropout(dropout),
-        dense(hidden_planes, planes, activation), Dropout(dropout))
+@functor PosEmbedding
+
+# Class tokens used by many vision transformer-like models
+struct CLSTokens
+  cls_token
 end
```

Comment:
```suggestion
struct CLSTokens{T}
  cls_token::T
end
```

+ docstring

Reply:
Yep, I've fixed the docstrings for the utility layers